src/orca/services/sevenbridges/hook.py

```diff
@@ -1,77 +1,23 @@
-from __future__ import annotations
-
-from functools import cached_property
-from typing import TYPE_CHECKING
-
-from airflow.exceptions import AirflowNotFoundException
-from airflow.hooks.base import BaseHook
-from sevenbridges import Api
-
-from orca.services.sevenbridges.config import SevenBridgesConfig
+from orca.services.base.hook import BaseOrcaHook
 from orca.services.sevenbridges.ops import SevenBridgesOps
 
-if TYPE_CHECKING:
-    from airflow.models.connection import Connection
-
 
-class SevenBridgesHook(BaseHook):
+class SevenBridgesHook(BaseOrcaHook):
     """Wrapper around SevenBridges client and ops classes.
 
-    This hook was inspired by the Asana Airflow provider package:
-    https://github.com/apache/airflow/blob/main/airflow/providers/asana/hooks/asana.py
+    Attributes:
+        conn_name_attr: Inherited Airflow attribute (e.g., "sbg_conn_id").
+        default_conn_name: Inherited Airflow attribute (e.g., "sbg_default").
+        conn_type: Inherited Airflow attribute (e.g., "sbg").
+        hook_name: Inherited Airflow attribute (e.g., "SevenBridges").
+
+    Class Variables:
```

Comment:
Good use of class variables here which is different from instance variables which are those Attributes above.